src/convert_to_dwc.R

```diff
@@ -43,13 +55,15 @@ DwcTaxonListEU %<>%
          kingdom = "Animalia", 
          phylum = "Artropoda" , 
          class = "Insecta") %>%
-  gbif_species_name_match(name_col = "scientificName", 
+  gbif_species_name_match(name_col = "scientificNameLocal", 
                           gbif_terms = c('usageKey', 'scientificName', 'rank', 
                                          'order', 'matchType','phylum', 
                                          'kingdom', 'genus','class',
                                          'confidence', 'synonym', 'status',
-                                         'family'))
-DwcTaxonListLocal %<>% 
+                                         'family'))%>%
+  mutate(taxonID = paste(usageKey, vdigest (scientificName, algo="md5"), sep=":"))
```

Comment:
just curious, what is this step doing?

Reply:
Create TaxonID

Reply:
so you create a unique identifier yourself using the md5 algorithm?

Reply:
+ GBIF usageKey